shared/src/main/scala/cps/runtime/SeqAsyncShift.scala

```diff
@@ -5,18 +5,17 @@ import scala.collection._
 import scala.reflect.ClassTag
 import scala.collection.mutable.ArrayBuilder
 
-class SeqAsyncShift[A, C[X] <: scala.collection.Seq[X] & scala.collection.SeqOps[X,C,C[X]], CA <: C[A]] 
+class SeqAsyncShift[A, C[X] <: scala.collection.Seq[X] & scala.collection.SeqOps[X,C,C[X]], CA <: C[A]]
        extends IterableOpsAsyncShift[A,C,CA]
            with PartialFunctionAsyncShiftBase[Int,A,CA]
    {
 
 
   // TODO: move to IndexedSeq
   def aggregate[F[_], B](c:C[A], m: CpsMonad[F])(
-        z: () => F[B])(seqop: (B, A) => F[B], combop: (B, B) ⇒ F[B]): F[B] =
-     c.aggregate[F[B]](z())(
+        z: () => F[B])(seqop: (B, A) => F[B]): F[B] =
+    c.foldLeft[F[B]](z())(
             (fb, a) => m.flatMap(fb)(b=>seqop(b,a)),
-            (fbx, fby) => m.flatMap(fbx)(bx=>m.flatMap(fby)(by=>combop(bx,by)))
```

Comment:
I guess I could simply leave the deprecated version as it was. It makes sense to keep the method consistent, and it is probably better to have the deprecated message directly from the compiler instead of relying on a comment, what do you think? Otherwise I could simply add a simple comment like this:
```scala
  // aggregate is deprecated since 2.13.0. Do not rely on this.
  @nowarn
```

Reply:
I can't understand why we need a warning. [Note that deprecation warnings on substituted methods in runtime are visible only to us [developers of dotty-cps-async], the end-user programmer will use the 'non-shifted' aggregate function and see a warning from the compiler regardless of the existence of our @nowarn).
 
If we define substitution for deprecated functions,  we already know that this function is deprecated (we can add deprecation annotation to our `aggregate`).   @nowarn comment can be something like
//. aggregate is deprecated, but we aim to perform the correct translation even if the programmer uses the deprecated method.

If we do a warning cleanup, then it better not show a deprecation warning for developers of dotty-cps-async because we already looked at the details and decided that we can do nothing with this Situation and don't need to reexamine this case again.  [Situation will be changed when `aggregate` has been deleted from the Scala standard library; after this, we also will have to delete our shifted version of the method].